src/main/java/com/google/starfish/services/UserService.java

```diff
@@ -3,15 +3,86 @@
 import java.sql.Connection;
 import java.sql.PreparedStatement;
 import java.sql.SQLException;  
-import java.sql.ResultSet;  
 import javax.sql.DataSource;
 
+enum Event {
+  FAVORITE, DOWNLOAD
+}
+
 /**
  * Service class for Users Table 
  */
 public class UserService extends TableService {
 
+  private String USERS = Table.USERS.getSqlTable();
+
+  // Points modifiers for different events
+  private final long FAVORITE_POINTS_MODIFIER = 3;
+  private final long DOWNLOAD_POINTS_MODIFIER = 5;
+
   public UserService() {
     super(Table.USERS);
   }
+
+  /** Increase user points when a user's note gets favorited */
+  public boolean increasePointsOnFavorite(DataSource pool, String userId) {
+    return increasePoints(pool, userId, Event.FAVORITE);
+  }
+
+  /** Increase user points when a user's note gets downloaded */
+  public boolean increasePointsOnDownload(DataSource pool, String userId) {
+    return increasePoints(pool, userId, Event.DOWNLOAD);
+  }
+
+  /** Variably increase user points based on event */
+  private boolean increasePoints(DataSource pool, String userId, Event event) {
+    if (userId == null) return false;
+    long pointsModifier = getPointsModifier(event);
```

Comment:
nit: prefer moving this above line 49 below to declare it only when it's needed (in case an error is thrown before even reaching that nested try, you'd be needlessly declaring this variable)

Reply:
Ahh that makes sense, nice catch!